charts/camunda-platform/values.yaml

```diff
@@ -976,6 +1000,270 @@ identity:
       # Ingress.tls.secretName defines the secret name which contains the TLS private key and certificate
       secretName: ""
 
+# Web-Modeler configuration of the Web Modeler subchart
+web-modeler:
+  # Enabled if true, the Web Modeler deployment and its related resources are deployed via a helm release
+  enabled: false
```

Comment:
We need to enable it during the tests to ensure it's at least deployable.

Reply:
Thanks, @aabouzaid! It would be great if we could implement this soon. We've just received a request from a customer last week who would like to try out Web Modeler and set it up in Kubernetes 🙂.

Regarding the Docker login: I think the recommendation from the Infra team is to store the credentials in Vault and import them in your CI workflow. That's also how we do it for Web Modeler:
https://github.com/camunda/web-modeler/blob/d8fb0938f38b0107ce669d4c87b6e04a9a76962f/.github/workflows/build-and-test.yml#L53-L73
I will ask the Infra team about this (and the account to be used for the Docker registry).